### PR TITLE
Metric name fix

### PR DESCRIPTION
**Pull request by ltv12**

### CODE REVIEW

File: src/applications/basic_analytics/eos_stb/general_driver.py [hunk 1]
```diff
@@ -17,26 +17,21 @@ class GeneralEosStbProcessor(BasicAnalyticsProcessor):
...
 
-    def __init__(self, configuration, schema):
-        super(GeneralEosStbProcessor, self).__init__(configuration, schema, "time")
+    def __init__(self, configuration):
+        BasicAnalyticsProcessor.__init__(self, configuration, GeneralEosStbProcessor.create_schema(),
+                                         timefield_name="time")
 
-    def _prepare_stream(self, read_stream):
-        return read_stream \
-            .select(from_json(read_stream["value"].cast("string"), self._schema).alias("json")) \
```
Comment (halfwhite) [inline src/applications/basic_analytics/eos_stb/general_driver.py]: Imports may need to be optimized after this.

File: src/applications/basic_analytics/eos_stb/odh_driver.py [hunk 2]
```diff
@@ -17,25 +17,18 @@ class EosStbOdhProcessor(BasicAnalyticsProcessor):
     - unique_count
     """
 
-    def __init__(self, configuration, schema):
-        super(EosStbOdhProcessor, self).__init__(configuration, schema, "ts")
+    def __init__(self, configuration):
+        BasicAnalyticsProcessor.__init__(self, configuration, EosStbOdhProcessor.create_schema())
```
Comment (halfwhite) [inline src/applications/basic_analytics/eos_stb/odh_driver.py]: Do you use default time field?

File: src/test/it/resources/basic_analytics/eos_stb/general/expected_result.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-{"@timestamp": "2017-07-26T15:00:00.000Z", "metric_name": "vagrant.eosdtv.lab5aobo.tst.heapp.stb.eos.acs.stb_id.count", "value": 3}
+{"@timestamp": "2017-07-26T15:00:00.000Z", "metric_name": "vagrant.eosdtv.lab5aobo.tst.heapp.stb.eos.acs.requests.count", "value": 3}
```
Comment (halfwhite) [inline src/test/it/resources/basic_analytics/eos_stb/general/expected_result.txt]: I would prefer singular form.

File: src/applications/basic_analytics/eos_stb/odh_driver.py (hunk 2 above)
Comment (ltv12) [reply src/applications/basic_analytics/eos_stb/odh_driver.py]: Yep

**Pull request merged**